src/main/query_result.cpp

```diff
@@ -186,9 +186,9 @@ std::string QueryResult::toString() {
 }
 
 void QueryResult::writeToCSV(
-    const std::string& fileName, char delimiter, char escapeCharacter, char newline) {
+    std::string_view fileName, char delimiter, char escapeCharacter, char newline) {
     std::ofstream file;
-    file.open(fileName);
+    file.open(std::string(fileName));
```

Comment:
This api has already been replaced by 'copy to' statement, so you can remove it.

Reply:
`COPY ... TO` (according to the docs) doesn't have a newline option, so it can't be used as a direct replacement in one of the tests (AlternateDelimCSVTest), though we could just change that test to expect `\n` as the newline character. Or maybe just remove those tests, or even update and add them to `copy_to_csv.test`.

Reply:
I'm going to merge this as-is. We can discuss removing writeToCSV later.